swag_exporter/setup_info.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "swag_export",
+  "version": "0.0.1",
+  "license": "License :: Other/Proprietary License",
```

Comment:
This is an open source public repo from our side. It isn't proprietary.
https://github.com/VoerEirAB/SwagExporter/blob/master/LICENSE

Reply:
so you mean i need to remove licence?

Reply:
Added apache license. Done!!